open-company/open-company-web
# Add NEW tag besides the comments summary

Part of: https://trello.com/c/CJahkpWQ

To test:
- open AP
- check a posts with 0, 1 and n comments
- [ ] all looking as before?
- now replace `oc.web.components.ui.comments-summary:56` with: `should-show-new-tag? true]`
- open AP
- [ ] do you see a NEW tag in the posts that have at least 1 comment?
- [ ] does it respect design and is it aligned?
- rollback the change
- merge
